pkg/scheduler/framework/plugins/migration/utils.go

```diff
@@ -88,3 +89,18 @@ func (p *PrioritiesStateData) Clone() framework.StateData {
 		Reference: p.Reference,
 	}
 }
+
+// PriorityMetadata returns priority metadata stored in CycleState.
+func PriorityMetadata(state *framework.CycleState) interface{} {
+	if state == nil {
+		return nil
+	}
+
+	var meta interface{}
+	if s, err := state.Read(PrioritiesStateKey); err == nil {
+		meta = s.(*PrioritiesStateData).Reference
+	} else {
+		klog.Errorf("reading key %q from CycleState, continuing without metadata: %v", PrioritiesStateKey, err)
```

Comment:
A nit: reading -> Error while reading ...

Reply:
-1
Errorf will already put an E in the log line. `while` is implied.

Reply:
I agree with Aldo on this one.